testsuite/integration/src/features/test_dws_states.feature

```diff
@@ -35,8 +36,8 @@ Feature: Data Workflow Services State Progression
             /bin/hostname
 
         When the job is run
-        And a Workflow is created for the job
-        Then the Workflow and job progress to the Proposal state
+        Then a Workflow has been created for the job
```

Comment:
What's the difference between Then/And in this case?

Reply:
As I was reading the list of statements, I thought it felt like an arbitrary thing, as far as when to switch from the "When" list to the "Then" list.  And so I pulled "Then" up a little higher in the list because that's how I felt that day.